llvm/lib/Target/AMDGPU/AMDGPUPostLegalizerCombiner.cpp

```diff
@@ -334,6 +336,19 @@ bool AMDGPUPostLegalizerCombinerImpl::matchRcpSqrtToRsq(
   return false;
 }
 
+void AMDGPUPostLegalizerCombinerImpl::applyOneFDivSqrtToRsq(
+    MachineInstr &MI, const Register &X) const {
+  // B.setInstrAndDebugLoc(MI);
```

Comment:
This should already be set so you can delete it - the combiner will automatically set the insertion point to the MI being looked at